network-api/networkapi/wagtailpages/factory/buyersguide.py

```diff
@@ -206,6 +206,8 @@ class Meta:
 
     title = Faker('sentence')
     hero_image = SubFactory(ImageFactory)
+    first_published_at = Faker('past_datetime', start_date='-30d', tzinfo=timezone.utc)
```

Comment:
Just some updates to the factory to make sure articles have a published date and description